app/main/system.ts

```diff
@@ -1,10 +1,10 @@
-import { Config } from "app/common/config"
-import log from "app/common/logging"
-import * as serializers from "app/common/serializers"
-import { Lifecycle } from "app/main/lifecycle"
-import * as components from "app/main/components"
+import log from "../common/logging"
+import {Config} from "../common/config"
+import {Lifecycle} from "./lifecycle"
+import * as serializers from "../common/serializers"
+import * as components from "../main/subsystems"
```

Comment:
Same here, I know you guys use intellij and it handles automatically the relative imports, but for me when I'm reading the code is very hard to know and refactor where things are